jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/api/Response.java

```diff
@@ -196,10 +199,37 @@ default void onContent(Response response, LongConsumer demand, ByteBuffer conten
         }
     }
 
+    /**
+     * Replacement for AsyncContentListener, intermediate between ContentListener and ContentSourceListener
+     */
+    interface SimpleContentSourceListener extends ContentSourceListener
```

Comment:
They are similar but not identical. The `onContent` signature changed `Runnable demander` instead of `Callback callback` and `Content.Chunk chunk` instead of `ByteBuffer content`.